src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -807,5 +807,17 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
                 }
             }
         }
+
+#nullable enable
+        public sealed override MethodSymbol? PartialImplementationPart => _property is SourcePropertySymbol { IsPartialDefinition: true, OtherPartOfPartial: { } other }
```

Comment:
>SourcePropertySymbol [](http://example.com/codeflow?start=87&length=20)

nit: the `SourcePropertySymbol` type check seems redundant (here and below)

Reply:
`_property` is a `SourcePropertySymbolBase` which doesn't have the members we are accessing within the pattern.